okhttp/src/main/kotlin/okhttp3/ConnectionPool.kt

```diff
@@ -152,5 +161,28 @@ class ConnectionPool internal constructor(
     @JvmField val backoffDelayMillis: Long = 60 * 1000,
     /** How much jitter to introduce in connection retry backoff delays */
     @JvmField val backoffJitterMillis: Int = 100,
+    /**
+     * The maximum number of concurrent calls per connection for a given address.
+     * Set this value to 1 to disable HTTP/2 connection coalescing
+     */
+    @JvmField val maximumConcurrentCallsPerConnection: Int? = null,
```

Comment:
nit: I think we use the `max` prefix elsewhere.
```suggestion
    @JvmField val maxConcurrentCallsPerConnection: Int? = null,
```